src/Powercord/plugins/pc-settings/components/GeneralSettings.jsx

```diff
@@ -118,7 +118,7 @@ module.exports = class GeneralSettings extends React.Component {
         </Category>
 
         <ButtonItem
-          note={'Removes everything stored in Discord\'s cache folder. This will make Discord slower, as all resources will have to be fetched again.'}
+          note={'Removes everything stored in Discord\'s cache folder. This will make temporarily make Discord slower, as all resources will have to be fetched again.'}
```

Comment:
Looks like you've put one too many "make"'s here. Let's fix that! 😊
```suggestion
          note={'Removes everything stored in Discord\'s cache folder. This will temporarily make Discord slower, as all resources will have to be fetched again.'}
```